src/lib/components/HttpErrors/HttpError.js

```diff
@@ -20,6 +20,7 @@ class HttpError extends Component {
               <Icon name={icon} size="massive" />
               <h1>{title}</h1>
               <h3>{message}</h3>
+              <h4>{errorId ? 'Error Id: ' + errorId : null}</h4>
```

Comment:
I would move the `h4` inside:
```suggestion
              {errorId ? `<h4>Error Id: ${errorId}</h4>` : null}
```